### DISCUSSION TITLE
Can we call it VK instead? (Upvote if you agree).

### DISCUSSION THREAD

**Discussion by componentscience** [valkey-io/valkey#290] [General]
Can we call VK instead of Valkey? Valkey sounds different. Thank you for this awesome project!

**Comment by stockholmux** [comment]
I think just "Valkey" is better.

 The more people see this and hear "Valkey" the message will spread faster. Shortening splits the name recognition. 

Also, VK is a Russian social media platform, so I'd like to avoid the confusion.
